### ISSUE TITLE
Dialogue tree underline formatting

### ISSUE THREAD

**Issue by PascalLadalle** [4ian/GDevelop#1603]
Yarn offers an underline option which is not (yet) supported by the BBCode object.
It should be greyed out or hidden for now.
![image](https://user-images.githubusercontent.com/25978830/77956561-61ef6900-72d2-11ea-85b4-7d019953cb7d.png)
In-game, either the text displays the tags, either it stops the text scrolling... not sure why one or the other.

GD5b89

**Comment by 4ian** [comment]
cc @blurymind what you have an idea about what to do?

**Comment by blurymind** [comment]
Yup, thats a limitation with pixi unfortunately. There is no styling for underlined text.

But I cant alter it in yarn, because other game engines that do support it will lose it too

**Comment by blurymind** [comment]
I could add an init option to yarn that hides the button

**Comment by blurymind** [comment]
I have added a known issues section to the wiki, to notify  users in advance about this external to gdevelop problem as well as others
http://wiki.compilgames.net/doku.php/gdevelop5/all-features/dialogue-tree
